src/views/OrderDetail.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-back-button slot="start" default-href="/home" />
+        <ion-title>{{ $t("PO External Order ID") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button @click="checkAllProducts">
+            <ion-icon :icon="checkboxOutline" />
+          </ion-button>
+          <ion-button>
+            <ion-icon :icon="arrowUndoOutline" />
+          </ion-button>
+        </ion-buttons>
+        
+        
+      </ion-toolbar>
+    </ion-header>
+
+    <ion-content :fullscreen="true">
+      <div class="header">
+        <div class="search">
+          <ion-searchbar> </ion-searchbar>
+        </div> 
+
+        <div class="filters">
+          <ion-item>
+            <ion-label>{{ $t("Buffer days") }}</ion-label>
+            <ion-input v-model="numberOfDays" type="text" placeholder="all items" /> 
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Order buffer") }}</ion-label>
+            <ion-input v-model="numberOfpieces" type="number" />
+          </ion-item>
+          <ion-item>
+            <ion-label>{{ $t("Catalog") }}</ion-label>
+            <ion-select v-model="catalog">
+              <ion-select-option>{{ $t("Backorder") }}</ion-select-option>
+              <ion-select-option>{{ $t("Preorder") }}</ion-select-option>
+            </ion-select>
+          </ion-item>
+          <ion-button expand="block" fill="outline" @click="apply">{{ $t("Apply") }}</ion-button>
+        </div> 
+      </div>  
+
+      <div v-for="id in getGroupList(orderItems)" :key="id" >
+        <div v-for="item in getGroupItems(id, orderItems)" :key="item">
+          <div class="list-header" >
+            <ion-label>{{ item.parentProductName }}</ion-label>
+            <ion-chip >
```

Comment:
You could use self closing `<div />` for empty space

Reply:
Used self closing <div />  for empty space.